.github/workflows/build.yml

```
@@ -133,3 +136,97 @@ jobs:
       - name: Setup tmate debug session
         uses: mxschmitt/action-tmate@v3
         if: env.RUN_TMATE
+  build:
+    # The AMI build process is an expensive test (in terms of time) so
+    # let's not run it unless the other jobs succeed.
+    needs:
+      - lint
+      - test
```

This is how it's done in [the cisagov/skeleton-python-library build job](https://github.com/cisagov/skeleton-python-library/blob/4ef2364701c0db074c423f740062b2d7c197bea8/.github/workflows/build.yml#L113). I'm fine if we want to switch to this style, but I'd like to use the same format for similar functionality whichever way we decide.

```suggestion
    needs: [lint, test]
```